rootfs-builder/ubuntu/config.sh

```diff
@@ -0,0 +1,29 @@
+# This is a configuration file add extra variables to
+#
+# Copyright (c) 2018  Yash Jain
+#
+# SPDX-License-Identifier: Apache-2.0
+# be used by build_rootfs() from rootfs_lib.sh the variables will be
+# loaded just before call the function. For more information see the
+# rootfs-builder/README.md file.
+#!/bin/bash
+OS_VERSION=${OS_VERSION:-16.04}
+PACKAGES="systemd iptables "
+
+# url to download rootfs from
+ARCHIVE_URL=${ARCHIVE_URL:-"http://archive.ubuntu.com/ubuntu/"}
+#ARCHIVE_URL=${ARCHIVE_URL:-"http://archive.debian.org/debian"}
```

Comment:
Please remove, or rework so that debian is also supported.

Reply:
Do you want me to add a new directory for supporting debian? The only change required will be to rename the OS_NAME param to "wheezy" everything else should work just fine.

Reply:
I appreciate they will be 99.99% identical, but yes, please create `rootfs-builder/debian/config.sh`. You could potentially put code common to ubuntu+debian somewhere (like maybe a function in `scripts/lib.sh`). Or you could just make the debian files source the ubuntu ones (or *vice versa*).